src/system-health/scripts/healthd

```diff
@@ -72,6 +73,8 @@ class HealthDaemon(DaemonBase):
             import sonic_platform.platform
             chassis = sonic_platform.platform.Platform().get_chassis()
             manager = HealthCheckerManager()
+            sysmon = Sysmonitor()
+            sysmon.system_service()
```

Comment:
sysmon.system_service is a infinite loop, the rest of the code in this function will never run.

Reply:
Taken care.